composer.json

```diff
@@ -12,11 +12,12 @@
         }
     ],
     "require": {
-        "php": "^5.5.9|^7.0",
+        "php": "^5.6.0|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0"
     },
     "require-dev": {
         "symfony/security-bundle": "^2.8|^3.0",
+        "phpunit/phpunit": "5.6.*",
         "symfony/phpunit-bridge": "^3.2",
```

Comment:
wouter, do we still need the bridge (for deprecations or such?) or can we remove this when we can use phpunit 5.6 for everything?